SteamReview/SteamReview/Steam.cpp

```diff
@@ -46,7 +53,8 @@ void FSteam::OpenMainMenu()
 void FSteam::OpenAddGame()
 {
 	//adding a new game
-	FGame Game = AddNewGame();
+	FGame Game;
+	AddNewGame(Game);
```

Comment:
Maybe you'd like to handle the scenario of the result of this function being false?